File: src/app/gkill/api/req_res/get_tag_history_by_tag_id_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/app/gkill/api/message"
	"github.com/mt3hr/gkill/src/app/gkill/dao/reps"
)

type GetTagHistoryByTagIDResponse struct {
	Messages []*message.GkillMessage `json:"messages"`

	Errors []*message.GkillError `json:"errors"`

	TagHistories []*reps.Tag `json:"tag_histories"`
}

File: src/client/classes/api/req_res/get-tag-history-by-tag-id-response.ts
'use strict'

import type { Tag } from '@/classes/datas/tag'
import { GkillAPIResponse } from '../gkill-api-response'

export class GetTagHistoryByTagIDResponse extends GkillAPIResponse {

    tag_histories: Array<Tag>

    constructor() {
        super()
        this.tag_histories = new Array<Tag>()
    }

}


